grease-cli/src/Grease/Cli.hs

```diff
@@ -452,6 +453,20 @@ simOpts = do
   simFsOpts <- fsOptsParser
   simInitPrecondOpts <- initPrecondOptsParser
   simBoundsOpts <- boundsOptsParser
+  simDumpCoverage <-
+    Opt.optional
+      ( Opt.strOption $
+          Opt.long "dump-coverage"
+            <> Opt.metavar "FILE"
+            <> Opt.help "Dump the addresses of executed instructions"
+      )
+  simDumpSectionMap <-
+    Opt.optional
+      ( Opt.strOption $
+          Opt.long "dump-sections"
+            <> Opt.metavar "FILE"
+            <> Opt.help "Dump a mapping from section indexes to addresses"
+      )
```

Comment:
Are these options that downstream projects might want to use in their CLIs? If so, might make sense to move these to the top level and export them, possibly wrapping them together in a struct with a single parser as is done with some other option groups.